mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -3302,6 +3349,37 @@ resource "google_compute_instance" "foobar" {
 `, instance)
 }
 
+func testAccComputeInstance_basic_resourceManagerTags(instance string) string {
```

Comment:
This doesn't appear to have set the new fields?

Reply:
Yes, I tested in my local provider and made sure the new fields worked fine, the reason why I did not add these new fields is because these fields are still in test stage and only allowlisted some specific projects. I will add these new fields once these fields are exposed to all projects

Reply:
Sorry, I missed this before because it was resolved. We can't move forward with this PR until the fields are made public, I'll follow up internally.